app/views/transferSummary.scala.html

```diff
@@ -0,0 +1,92 @@
+@import java.util.UUID
+@import viewsapi.FormFunctions._
+@import helper.{inputRadioGroup => _, _}
+
+@(consignmentId: UUID, transferSummaryForm: Form[TransferSummaryData])(implicit request: RequestHeader, messages: Messages)
+@main(Messages("transferSummary.header")) {
+@defining(play.core.PlayVersion.current) { version =>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <a href="@routes.DashboardController.dashboard()" class="govuk-back-link">Back</a>
+            @progressIndicator(Messages("transferSummary.progress"))
+            <h1 class="govuk-heading-xl">@Messages("transferSummary.title")</h1>
+            <dl class="govuk-summary-list">
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        @Messages("transferSummary.seriesReference")
+                    </dt>
+                    <dd class="govuk-summary-list__value summary-list__right-column">
+
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        @Messages("transferSummary.consignmentReference")
```

Comment:
Could you remove this row from the table for now? We won't be able to fill in the consignment reference for a while, so it's better not to show the row at all so it doesn't confuse users. See the note at the end of the acceptance criteria on [TDR-223](https://national-archives.atlassian.net/browse/TDR-223).

Reply:
Sure, will do. Sorry, I missed this.